srcpkgs/python-pywatchman/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-pywatchman'
+pkgname=python-pywatchman
+version=1.4.1
+revision=1
+wrksrc="pywatchman-${version}"
+build_style=python-module
+only_for_archs="x86_64 x86_64-musl"
+pycompile_module="pywatchman"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+makedepends="curl"
+depends="python watchman"
+short_desc="Connect and query Watchman to discover file changes (Python 2)"
```

Comment:
It is preferred to use `(Python2)` 

which can be replaced later on in `short_desc="${short_desc/Python2/Python3}"`